opencti-platform/opencti-front/src/private/components/dataEntities/Entities.js

```diff
@@ -0,0 +1,366 @@
+import React, { Component } from 'react';
+import * as PropTypes from 'prop-types';
+import { withRouter } from 'react-router-dom';
+import * as R from 'ramda';
+import { QueryRenderer as QR } from 'react-relay';
+import Typography from '@material-ui/core/Typography';
+import { QueryRenderer } from '../../../relay/environment';
+import QueryRendererDarkLight from '../../../relay/environmentDarkLight';
+import {
+  buildViewParamsFromUrlAndStorage,
+  convertFilters,
+  saveViewParameters,
+} from '../../../utils/ListParameters';
+import inject18n from '../../../components/i18n';
+import CyioListCards from '../../../components/list_cards/CyioListCards';
+import CyioListLines from '../../../components/list_lines/CyioListLines';
+// import RisksCards, {
+//   risksCardsQuery,
+// } from './risks/RisksCards';
+// import RisksLines, {
+//   risksLinesQuery,
+// } from './risks/RisksLines';
+import EntitiesCreation from './data/EntitiesCreation';
+import Security, { KNOWLEDGE_KNUPDATE } from '../../../utils/Security';
+import { isUniqFilter } from '../common/lists/Filters';
+import EntitiesDeletion from './data/EntitiesDeletion';
+import ErrorNotFound from '../../../components/ErrorNotFound';
+import { toastSuccess, toastGenericError } from '../../../utils/bakedToast';
+
+class Entities extends Component {
+  constructor(props) {
+    super(props);
+    const params = buildViewParamsFromUrlAndStorage(
+      props.history,
+      props.location,
+      'view-data',
+    );
+    this.state = {
+      sortBy: R.propOr('', 'sortBy', params),
+      orderAsc: R.propOr(true, 'orderAsc', params),
+      searchTerm: R.propOr('', 'searchTerm', params),
+      view: R.propOr('cards', 'view', params),
+      filters: R.propOr({}, 'filters', params),
+      openExports: false,
+      numberOfElements: { number: 0, symbol: '' },
+      selectedElements: null,
+      selectAll: false,
+      openDataCreation: false,
+      selectedDataEntity: '',
+    };
+  }
+
+  saveView() {
+    // this.handleRefresh();
+    saveViewParameters(
+      this.props.history,
+      this.props.location,
+      'view-data',
+      this.state,
+    );
+  }
+
+  handleChangeView(mode) {
+    this.setState({ view: mode }, () => this.saveView());
+  }
+
+  handleSearch(value) {
+    this.setState({ searchTerm: value }, () => this.saveView());
+  }
+
+  handleSort(field, orderAsc) {
+    this.setState({ sortBy: field, orderAsc }, () => this.saveView());
+  }
+
+  handleToggleExports() {
+    this.setState({ openExports: !this.state.openExports });
+  }
+
+  handleToggleSelectAll() {
+    this.setState({ selectAll: !this.state.selectAll, selectedElements: null });
+  }
+
+  handleClearSelectedElements() {
+    this.setState({ selectAll: false, selectedElements: null });
+  }
+
+  handleRiskCreation() {
+    this.setState({ openDataCreation: true });
+  }
+
+  handleRefresh() {
+    this.props.history.push('/data/entities');
+  }
```

Comment:
@sachin755 looks to not be used. If that is intentional, would you please remove it?